### PR TITLE
Update api

### PR DESCRIPTION
**Pull request by mhpicel**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
```
Comment (manzaloros) [inline config.js:1]: Let me know if you need any help getting these to work on your server!

File: db/generateData.js [hunk 2]
```diff
@@ -32,11 +31,11 @@ var writeData = (nrecords) => {
       var randInd = Math.floor(Math.random() * imgUrls.length);
       var randUrl = imgUrls[randInd];
       if (listingId === nrecords) {
-        writer.write([globalId, imageId, listingId, randUrl]);
+        writer.write([listingId, imageId, randUrl]);
```
Comment (manzaloros) [inline db/generateData.js:34]: Did you not have to use the drain event?

**Review by manzaloros — APPROVED** [review]
Looking good, Mark. Does your CRUD work with your new database as well?

File: db/generateData.js (hunk 2 above)
Comment (mhpicel) [reply db/generateData.js:34]: it's in there at the bottom

**Comment by mhpicel** [comment]
> Looking good, Mark. Does your CRUD work with your new database as well?

yep!

**Pull request merged**